benchmarks/benchmark-ab.py

```diff
@@ -505,11 +507,16 @@ def generate_csv_output():
     artifacts["TS latency P90"] = extract_entity(data, "90%", -1)
     artifacts["TS latency P99"] = extract_entity(data, "99%", -1)
     artifacts["TS latency mean"] = extract_entity(data, "Time per request:.*mean\)", -3)
-    artifacts["TS error rate"] = (
-        int(artifacts["TS failed requests"]) / execution_params["requests"] * 100
-    )
+    if isinstance(artifacts["TS failed requests"], type(None)):
+        artifacts["TS error rate"] = None
```

Comment:
artifacts["TS error rate"] should be number

Reply:
changed to 0.0